src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -0,0 +1,974 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <array>
+#include <functional>
+#include <utility>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "../custom_node.hpp"
+#include "../custom_node_library_manager.hpp"
+#include "../dl_node.hpp"
+#include "../entry_node.hpp"
+#include "../exit_node.hpp"
+#include "../node_library.hpp"
+#include "../pipelinedefinition.hpp"
+#include "test_utils.hpp"
+
+using namespace ovms;
+using namespace tensorflow;
+using namespace tensorflow::serving;
```

Comment:
In general this is bad practice - using directives - especially when using several at once. Maybe just add using for PredictRequest/Response as I suspect those are the only ones needed?

Reply:
Done